shared/src/main/java/io/pravega/shared/segment/StreamSegmentNameUtils.java

```diff
@@ -43,6 +47,11 @@
      */
     private static final String TRANSACTION_DELIMITER = "#transaction.";
```

Comment:
Isn't this now deprecated?

Reply:
It is not understood by segmentstore. segmentstore does not care about how we create our segment-names as long as they are unique. 
Controller will continue to create transaction segments that follow this naming convention and from clients perspective nothing changes about the way it handles transactions. Hence this delimiter is used by both of them as before.